scripts/docker/keycloak/master-realm.json

```diff
@@ -963,6 +963,7 @@
       "consentRequired" : false,
       "config" : {
         "user.attribute" : "foo",
+        "id.token.claim": "true",
```

Comment:
Users can have a LOT of roles, we deliberately do not add the roles to the tokens, otherwise the header size limit could quickly be reached (and requests would become slow).
If this admin UI should only be accessible to users with the admin role, we could think about different solutions, like an endpoint to check if the users has the admin role, or adding just a specific role to the token (if that is possible).

Reply:
Yes, I did notice that, however this would only be needed for the id token, and not the access token that is used in the header of the requests.

Reply:
Of course other solutions can be thought of too, however I am wondering if there will be a need to hide/show some buttons etc. based on user roles for other users too later on, so it might not just be about who's an admin and who's not.

Reply:
> Yes, I did notice that, however this would only be needed for the id token, and not the access token that is used in the header of the requests.

I missed that detail, for the id token it should indeed be fine to add the roles.

But I also noticed that the roles are still added to the access token in the compose setup, it seems we only changed that in our internal config. That will at some point become an issue when more repositories are added.